### PR TITLE
Bump node-sass to v6.0.0

### PR DESCRIPTION
**Pull request by sstrubberg**
To support Node v16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "chalk": "^2.3.0",
     "lodash": "^4.17.11",
-    "node-sass": "^4.8.3",
+    "node-sass": "^6.0.0",
```
Comment (xzyfer) [inline package.json:29]: There's no reason to drop support for old versions.
```suggestion
    "node-sass": ">=4.8.3 && <7",
```

**Comment by cyrilverloop** [comment]
Hi,

Node 16 support would be appreciated.
Is there any help needed here ?

**Comment by CodePhilanthropist** [comment]
man! this is where I am having headaches. please merge this pr to solve the v16 issue

**Comment by Anoesj** [comment]
From README of package `node-sass`:

> Warning: LibSass and Node Sass are deprecated. While they will continue to receive maintenance releases indefinitely, there are no plans to add additional features or compatibility with any new CSS or Sass features. Projects that still use it should move onto Dart Sass.

So if transitioning to Dart Sass (`sass`) is the way to go, `node-sass` shouldn't break this package when you're using the `sass.compiler = require('sass');` option (because you're not actually using `node-sass` in that scenario). So why is `node-sass` a dependency at all? Why not a peer dependency or optional dependency or something? I'm not sure what would be the best solution, all I know is it's very frustrating that `gulp-sass` does not work on Node 16, even if you do not use `node-sass` at all.

**Comment by mxmason** [comment]
When I encountered this issue myself, I saw a few more things need to be done to ship this fix – see #802.

One of my open-source teams uses `gulp-sass`, and its current dependency graph is a blocker for anyone on an M1 Mac, since M1s can only run Node 16. Please consider my PR, or any PR which updates `node-sass`, because this is a significant blocker for M1 devices.

**Comment by xzyfer** [comment]
Thanks all, close this to focus efforts on #802 which is more rounded at this point in time.

**Pull request closed without merging**